test/e2e/utils.py

```diff
@@ -88,6 +105,142 @@ def submit_sleep_job(
     return job
 
 
+def submit_job_from_bundle(
+    deadline_client: DeadlineClient,
+    farm: Farm,
+    queue: Queue,
+    bundle_path: str,
+    template_file_name: str = "template",
+    job_attachments_file_system: str = "COPIED",
+    queue_parameter_definitions: List[dict] = [],
+    max_retries_per_task: Optional[int] = None,
+) -> Job:
+    bundle_path = os.path.normpath(bundle_path)
+    LOG.info(f"Submitting bundle {bundle_path} to farm {farm.id} and queue {queue.id}")
+    yaml_path = f"{bundle_path}/{template_file_name}.yaml"
+    json_path = f"{bundle_path}/{template_file_name}.json"
+    if os.path.isfile(yaml_path):
+        with open(yaml_path) as f:
+            job_template = yaml.safe_load(f.read())
+    elif os.path.isfile(json_path):
+        with open(json_path) as f:
+            job_template = json.loads(f.read())
+    else:
+        LOG.error(
+            f"Was expecting to find either template.yaml or template.json in directory {bundle_path} but found none."
```

Comment:
```suggestion
            f"Was expecting to find either {template_file_name}.yaml or {template_file_name}.json in directory {bundle_path} but found none."
```

You could also just directly use `yaml_path` and `json_path` variables in here